geode-assembly/build.gradle

```diff
@@ -144,6 +177,13 @@ dependencies {
   upgradeTestRuntime 'org.apache.httpcomponents:httpclient:' + project.'httpclient.version'
   upgradeTestRuntime project(':extensions:session-testing-war')
 
+  //Web servers used for session module testing
+  webServerTomcat6 group: 'apache', name: 'tomcat', version: '6.0.37', ext: 'zip'
+  webServerTomcat73 group: 'org.apache.tomcat', name: 'tomcat', version: '7.0.73', ext: 'zip'
+  webServerTomcat79 group: 'org.apache.tomcat', name: 'tomcat', version: '7.0.79', ext: 'zip'
+  webServerTomcat8 group: 'org.apache.tomcat', name: 'tomcat', version: '8.5.15', ext: 'zip'
+  webServerTomcat9 group: 'org.apache.tomcat', name: 'tomcat', version: '9.0.12', ext: 'zip'
+  webServerJetty group: 'org.eclipse.jetty', name: 'jetty-distribution', version: '9.4.8.v20171121', ext: 'zip'
```

Comment:
For consistency, these versions should make sure of the project properties present in `dependency-versions.properties`, e.g.,
```
  webServerJetty group: 'org.eclipse.jetty', name: 'jetty-distribution', version: project.'jetty.version', ext: 'zip'
```

I note that neither of these Tomcat7 versions match that which we define in our dependency versions file, which should be updated to reflect actual consumption.

Reply:
Good point. I switched to using depency-versions.properties and now we are using consistent versions of tomcat. Thanks!